todo-app/src/Components/todoList.js

```diff
@@ -0,0 +1,74 @@
+//jshint esnext: true
+
+import React, { Component, Fragment } from 'react'
+
+import TodoItem from './todoItem';
+import './styles/todoList.css';
+
+export default class TodoList extends Component {
+
+    state = {
+        toBeUpdatedItem: '',
+        toBeUpdatedDeadline: ''
+    }
+
+    handleCancelEdit = () => {
+        this.props.handleToggleEdit();        
+    }
+
+    handleToggleEdit = id => {
+        const todoItem = this.props.todos.find(item => item.id === id);
+        this.setState({ 
+            toBeUpdatedDeadline: todoItem.deadline,
+            toBeUpdatedItem: todoItem.text
+        });
+        this.props.handleToggleEdit(id);
+    }
+
+    handleUpdate = (id) => {
+        const { toBeUpdatedItem, toBeUpdatedDeadline } = this.state;
+        this.props.handleUpdate(id, toBeUpdatedItem, toBeUpdatedDeadline);
+    }
+
+    handleChangeItem = event => {
+        this.setState({ toBeUpdatedItem: event.target.value })
+    }
+
+    handleChangeDeadline = event => {
+        this.setState({ toBeUpdatedDeadline: event.target.value })        
+    }
+    
+    renderAllTodos = () => {
+        const todoItems = this.props.todos;
+        return todoItems.map(item => {
+            return (
+            <Fragment key={item.id}>
```

Comment:
Fragments were introduced in React 16 to let you return an array of children. Since you have a single child here, just give the `key` directly to your `TodoItem` component.